### PR TITLE
Minor changes

### PR DESCRIPTION
**Pull request by hemerem-ch**
-update business page back link
- update address validation
- added business name/name to list and manual page

### CODE REVIEW

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

File: src/main/views/features/unincorporated/business-address-auto-lookup/auto-lookup-address.njk [hunk 1]
```diff
@@ -4,7 +4,12 @@
 {% block main_content %}
   <div class="govuk-width-container">
     <form action="" method="POST">
-      {% include "partials/business_name.njk" %}
+      {% if businessName %}
```
Comment (swalke-ch) [inline src/main/views/features/unincorporated/business-address-auto-lookup/auto-lookup-address.njk:7]: Has businessName been set in the corresponding controller ?

File: src/main/views/features/unincorporated/business-address-auto-lookup/business-address-list.njk [hunk 2]
```diff
@@ -18,11 +16,16 @@
         } %}
         {% set ITEMS = ITEMS.concat(addressObject) %}
     {% endfor %}
-    
     {# Form section #}
     <form action="" method="POST">
-        {% include "partials/business_name.njk" %}
-        <div class="govuk-form-group">
+        {% if businessName %}
```
Comment (swalke-ch) [inline src/main/views/features/unincorporated/business-address-auto-lookup/business-address-list.njk:21]: Has businessName been set in the corresponding controller ?

**Review by swalke-ch — APPROVED** [review]

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

**Review by sampankumar — APPROVED** [review]

**Pull request merged**